packages/foundations/scss/_init.scss

```
@@ -32,7 +32,12 @@ html {
 }
 
 // Make sure every elements has the current color from container above
-body {
+body,
+button,
+input,
+textarea,
+summary,
+select {
```

```suggestion
button,
input,
textarea,
summary,
select {
    color: inherit;
}
body,
```

I'm struggling with the aspect whether we would be more flexible with an approach like this.

---

fixed